webpack.config.dev.js

```diff
@@ -22,8 +22,24 @@ module.exports = {
   module: {
     loaders: [{
       test: /\.js$/,
-      loaders: ['babel'],
-      include: path.join(__dirname, 'source')
+      include: path.join(__dirname, 'source'),
+      loader: 'babel',
+      query: {
```

Comment:
I've deleted the `.babelrc`. .babelrc doesn't work with react-transform like I tested.